command/configtest_test.go

```diff
@@ -0,0 +1,105 @@
+package command
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/mitchellh/cli"
+)
+
+func TestConfigTestCommand_implements(t *testing.T) {
+	var _ cli.Command = &ConfigTestCommand{}
+}
+
+func TestFailOnEmptyFile(t *testing.T) {
```

Comment:
These tests should use a prefix that relates it to the configtest command so we can avoid collisions later on.